### PR TITLE
fix lava testnet to testnet-2

### PR DESCRIPTION
**Pull request by omerlavanet**
the current entry is outdated

### CODE REVIEW

File: testnets/lavatestnet2/chain.json [hunk 1]
```diff
@@ -56,91 +56,9 @@
     ],
     "genesis": {
       "name": "v0.3.0",
-      "genesis_url": "https://raw.githubusercontent.com/lavanet/lava-config/main/testnet-1/genesis_json/genesis.json"
+      "genesis_url": "https://raw.githubusercontent.com/lavanet/lava-config/main/testnet-2/genesis_json/genesis.json"
     },
     "versions": [
```
Comment (JeremyParish69) [inline testnets/lavatestnet2/chain.json:61]: Please add the current version (v0.32.0) to the `versions` [] array

Comment (omerlavanet) [reply testnets/lavatestnet2/chain.json:61]: thanks, done

**Comment by JeremyParish69** [comment]
Looking over this, I think we need to revert this PR. I think this was meant to be an archived chain with id lava-testnet-1, whereas lava-testnet-2 has chain_name: "lavatestnet". I think most the properties here are meant to be reverted, the status set to killed, and the chain_name changed to lavatestnet1 to properly archive this as having been abandoned post a hard-fork--the one leading to the creating of lava-testnet-2.
Does this sound acceptable? For now, to correct the duplicate chain-id problem, I need to at least set the chain_id back to lava-testnet-1, but it aught to be fully archived.

**Comment by omerlavanet** [comment]
> Looking over this, I think we need to revert this PR. I think this was meant to be an archived chain with id lava-testnet-1, whereas lava-testnet-2 has chain_name: "lavatestnet". I think most the properties here are meant to be reverted, the status set to killed, and the chain_name changed to lavatestnet1 to properly archive this as having been abandoned post a hard-fork--the one leading to the creating of lava-testnet-2. Does this sound acceptable? For now, to correct the duplicate chain-id problem, I need to at least set the chain_id back to lava-testnet-1, but it aught to be fully archived.

yes, makes sense. we can kill the lava-testnet-1 and create a new lava-testnet-2, what triggered the need for the change?

**Comment by JeremyParish69** [comment]
The trigger for this was the addition of automation to detect chain-id conflicts. The chain-id for multiple chains had lava-testnet-2 which will no longer be allowed at the chain registry.
I'd like to essentially revert the changes in this PR to restore the version history and other details of lavatestnet1, plus set its status to 'killed'(, assuming it to be the pre-fork chain of lava-testnet-2), and to update the chain_name for lavatestnet2 to just 'lavatestnet'. This way, no matter which version of Lava Testnet is current, it will always keep the chain_name: lavatestnet.  This is how other chains have handled archiving pre-fork chains.

**Comment by JeremyParish69** [comment]
This is the PR I propose to correct the handling of the pre-form chain instance:
https://github.com/cosmos/chain-registry/pull/4201

**Pull request merged**